utils/scraper/processGeneralData.js

```diff
@@ -20,48 +20,39 @@ const parseDocument = async pdfPath => {
   }
 }
 
-const extractValue = (data, regexPattern) => {
-  const regex = new RegExp(regexPattern) // create a new regular expression object using the pattern provided
-  const match = regex.exec(data[3]) // regex.exec executes the regular expression regex on the fourth element of the data array
-  if (match && match[1]) {
-    return match[1].trim()
-  } else {
-    return "*"
```

Comment:
I wonder if we should still return some kind of identifiable if something goes wrong? It's helped us a lot debugging so far